src/morph/src/Server.rei

```diff
@@ -1,21 +1,22 @@
 /**
 A [handler] takes a [Request.t] and returns a [Response.t] wrapepd in a [Lwt.t]
-
-See {{: https://rgrinberg.github.io/opium/opium_core/index.html#services} [Opium_core.Service.t]} for documentation on Services.
 */
-type handler = Opium_core.Service.t(Request.t, Response.t);
+type handler('req_body) = Request.t('req_body) => Lwt.t(Response.t);
 
 /**
 A [middleware] takes a [handler] and returns a [handler].
-
-See {{: https://rgrinberg.github.io/opium/opium_core/index.html#filters} [Opium_core.Filter.t]} for documentation on Filters.
 */
-type middleware = Opium_core.Filter.simple(Request.t, Response.t);
+type middleware('req_body) =
+  (Request.t('req_body) => Lwt.t(Response.t), Request.t('req_body)) =>
```

Comment:
The relationship between middleware and handler will be more obvious if you replace `Request.t('req_body) => Lwt.t(Response.t)` with `handler('req_body)`. Previously I scratched my head about it.

Reply:
I thought I did that change, might have missed it in the interface. I also wanted to move middleware and handler to a separate module, any thoughts about a name?

Reply:
I think it's fine here. The module is small!